packages/lesswrong/components/revisions/RevisionSelect.tsx

```diff
@@ -84,29 +85,31 @@ const RevisionSelect = ({ revisions, getRevisionUrl, onPairSelected, loadMorePro
           <tr key={rev.version} className={classes.revisionRow}>
             <td>
               <LWTooltip title={<div>Select as the <em>first</em> revision to compare</div>}>
-                <Radio
-                  className={classNames(classes.radio, {[classes.checked]: i===beforeRevisionIndex, [classes.radioDisabled]: beforeDisabled})}
-                  disabled={beforeDisabled}
-                  checked={i===beforeRevisionIndex}
-                  onChange={(ev, checked) => {
-                    if (checked) {
-                      setBeforeRevisionIndex(i);
-                    }
-                  }}
-                />
+                  <Radio
```

Comment:
this whitespacing looks wrong to me? (not a huge deal but confused why it's here)

Reply:
Yep, fixed